src/utils/components/groupHomeAdminLinks.story.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { MOCK_GROUP } from 'meetup-web-mocks/lib/api';
+
+import GroupHomeAdminLinks from './GroupHomeAdminLinks';
+
+storiesOf('GroupHomeAdminLinks', module)
+	.add('default: isAdmin={false}', () => (
```

Comment:
Add our standard decorator by putting `.addDecorator(decorateWithBasics)` before this line.
This will give you icons, and center the hint text you added. `decorateWithBasics` can be imported from `src/utils/decorators.jsx`